blocks/api/simple-dom.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { forEach } from 'lodash';
+
+export function createSimpleNode( node, filter = Array ) {
```

Comment:
Can we add some JSDoc for these functions?

Reply:
Sure it's a WIP.